test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_UI_SetGlobalProperties_WARNINGS.lua

```diff
@@ -0,0 +1,161 @@
+local ServerAddress = commonFunctions.read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)		
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,_)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+        :Times(1)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_SetGlobalProperties_WARNINGS()
+	--mobile side: sending SetGlobalProperties request
+	local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+	{
+		menuTitle = "Menu Title",
+		timeoutPrompt = 
+		{{
+				text = "Timeout prompt",
+				type = "TEXT"
+			}},
+		vrHelp = 
+		{{
+				position = 1,
+				image = 
+				{
+					value = "action.png",
+					imageType = "DYNAMIC"
+				},
+				text = "VR help item"
+			}},
+		menuIcon = 
+		{
+			value = "action.png",
+			imageType = "DYNAMIC"
+		},
+		helpPrompt = 
+		{{
+				text = "Help prompt",
+				type = "TEXT"
+			}},
+		vrHelpTitle = "VR help title",
+		keyboardProperties = 
+		{
+			keyboardLayout = "QWERTY",
+			language = "EN-US"
+		}
+	})
+	--hmi side: expect TTS.SetGlobalProperties request
+	EXPECT_HMICALL("TTS.SetGlobalProperties",
+	{
+		timeoutPrompt = 
+		{{
+				text = "Timeout prompt",
+				type = "SAPI_PHONEMES"
+			}},
+		helpPrompt = 
+		{{
+				text = "Help prompt",
+				type = "TEXT"
+			}},
+		appID = self.applications["Test Application"]
+	})
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, "TTS.SetGlobalProperties", "SUCCESS", {})
+	end)
+	
+	--hmi side: expect UI.SetGlobalProperties request
+	EXPECT_HMICALL("UI.SetGlobalProperties",
+	{
+	    vrHelpTitle = "VR help title",
+	    vrHelp = 
+		{{
+				position = 1,
+				image = 
+				{
+					imageType = "STATIC",
+					value = storagePath .. "action.png"
```

Comment:
@vvvakulenko, please make verification of 'value' to be checked as sub-string, because at the moment the test has an error:
You can use as example:
if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."action.png)" .. "$") == nil ) then
-- print some error
end